subworkflows/local/differential/main.nf

```diff
@@ -72,7 +77,9 @@ workflow DIFFERENTIAL {
     //     .mix(DESEQ2_DIFFERENTIAL.out.results)
 
     emit:
-    results   = ch_results
-    adjacency = ch_adjacency
-    // TODO add filtered results channel
+    results_pairwise          = ch_results_pairwise
+    results_pairwise_filtered = ch_results_pairwise_filtered
+    results_genewise          = ch_results_genewise
```

Comment:
I believe `ch_results_genewise` should also be filled with something. IMO it's quite unintuitive that the `ch_results_genewise` would be empty while `ch_results_genewise_filtered` is not.

Reply:
i have added now :)